docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,109 @@
-Assessment
-==========
+Introduction
+============
 
-Metrics
--------
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
 
-.. currentmodule:: fairlearn.metrics
+The steps of the assesment are as follows:
+1. Identify harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
+
+1. Identify harms
+-----------------
+
```

Comment:
We may want to add a reference to the types of harms in the previous user guide section 
https://fairlearn.org/main/user_guide/fairness_in_machine_learning.html#types-of-harms

Reply:
Done!